earth2studio/models/px/sfno.py

```diff
@@ -254,24 +262,58 @@ def load_model(
         PrognosticModel
             Prognostic model
         """
-        model = load_model_package(package)
+
+        # Makani load_model_package
+        path = package.resolve("config.json")
+        params = ParamsBase.from_json(path)
+
+        # Set global_means_path and global_stds_path
+        params.global_means_path = package.resolve("global_means.npy")
+        params.global_stds_path = package.resolve("global_stds.npy")
+        params.in_channels = np.arange(len(variables))
+        params.out_channels = np.arange(len(variables))
+
+        LocalPackage._load_static_data(package, params)
+
+        # assume we are not distributed
+        # distributed checkpoints might be saved with different params values
+        params.img_local_offset_x = 0
+        params.img_local_offset_y = 0
+        params.img_local_shape_x = params.img_shape_x
+        params.img_local_shape_y = params.img_shape_y
+
+        # get the model
+        model = model_registry.get_model(params, multistep=False).to(device)
+
+        # Load checkpoint
+        best_checkpoint_path = package.get(LocalPackage.MODEL_PACKAGE_CHECKPOINT_PATH)
+        checkpoint = torch.load(
```

Comment:
could probably `torch.get_default_device()` as the `device` object to skip the need to add a parameter to the load function.

probably more consistent with other load methods that way. but im not too worried since this is a big work around for api breaks.

Reply:
I think we should think about this more in the future because I am not convinced that relying on `get_default_device` is a great solution (particularly here).